plugins/io.sarl.lang/src/io/sarl/lang/jvmmodel/SARLJvmModelInferrer.java

```diff
@@ -2283,7 +2283,7 @@ protected void translateSarlFormalParameters(
 	 */
 	protected List<String> translateSarlFormalParametersForSyntheticOperation(JvmExecutable owner, JvmGenericType actionContainer,
 			boolean varargs, List<InferredStandardParameter> signature) {
-		JvmFormalParameter lastParam = null;
```

Comment:
Move the declaration for obtaining `final JvmFormalParameter lastParam = this.typesFactory.createJvmFormalParameter();`